### PR TITLE
[Merged by Bors] - TY-3018 improve justfile for db

### PR DESCRIPTION
**Pull request by rustonaut**
- adds a easy way to setup a temp-file sqlite db for usage with sqlx-cli, and a easy way to calling `sqlx migrate` on it
- run clippy with storage feature enabled
- fix missed clippy lints

### CODE REVIEW

**Review by finiteprods — APPROVED** [review]
lgtm

File: justfile [hunk 1]
```diff
@@ -105,6 +105,7 @@ _codegen-order-workaround:
 # Checks rust code, fails on warnings on CI
 rust-check: _codegen-order-workaround
     cd "$RUST_WORKSPACE"; \
+    cargo clippy --all-targets --features="storage" --locked; \
```
Comment (janpetschexain) [inline justfile:108]: 🔧 this is a duplicate of l.110
```suggestion
```

Comment (rustonaut) [reply justfile:108]: :rocket: a notorious "no error, no warning" merge/rebase failure :scream:

**Review by janpetschexain — APPROVED** [review]

**Comment by rustonaut** [comment]
bors merge

**Pull request closed without merging**